### PR TITLE
Validate SOUL

### PR DESCRIPTION
**Pull request by iskeceli**
SOUL token add

# Validate [SOUL](https://solscan.io/token/F6weWmuc1vwdL4u38Ro9jKXHEMjP9BoNdWMF5o5TvtJf)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/A2ZDAO/status/1684116507977195520
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

 Mint address: F6weWmuc1vwdL4u38Ro9jKXHEMjP9BoNdWMF5o5TvtJf
 Symbol: SOUL
 Token Decimals: 9
 Token icon: https://rkmp2iorabaal3oqws5jj233ruddww4itlhmkrigin5nctwm4isa.arweave.net/ipj9IdEAQAXt0LS6lOt7jQY7W4iazsVFBkN60U7M4iQ
 On-chain metadata matches

Community tweets: https://twitter.com/A2ZDAO/status/1684116507977195520

### CODE REVIEW

**Comment by iskeceli** [comment]
Mint address: F6weWmuc1vwdL4u38Ro9jKXHEMjP9BoNdWMF5o5TvtJf
Symbol: SOUL
Token Decimals: 9
Token icon: https://rkmp2iorabaal3oqws5jj233ruddww4itlhmkrigin5nctwm4isa.arweave.net/ipj9IdEAQAXt0LS6lOt7jQY7W4iazsVFBkN60U7M4iQ
On-chain metadata matches

Community tweets: https://twitter.com/A2ZDAO/status/1684116507977195520

File: validated-tokens.csv [hunk 1]
```diff
@@ -753,3 +753,4 @@ JEFF,JEFF,FkCaFsprX7gySagFoQPHNbe9MRkjrvh21cokJo6C1e2T,6,https://arweave.net/Pi-
 DIG Gold,DIG,HQMBNWptHTTkGY5FDAypqNqQzx9jF6cNJEZyrxoMMAVs,9,https://shdw-drive.genesysgo.net/6Y4jCS21YEVcEiL5qfifYRk6gQ1ozkWs8LsmSTAJeQR6/SPL_TOKEN.png,true
 ASTRALIS,ASTRALIS,ASTRALvKjGK2xk2pamjMBU5dav5cEQa6zpKCP6FZ7BAJ,6,https://shdw-drive.genesysgo.net/AcG1tjtJ3vR1XzSCcVArheGLrwmMsphqExa6VodLymGF/astralis_1500x1500_8bit.png,true
 GEM,GEM,https://solscan.io/token/E7BGDtpNXUTqPNbZxKHiLowgLddiAeuKcByD7tSnfYWD,5,https://bafkreifpm7rbwohp4yys2guyibfzqbctu3bist6zetmyq456om2rqvk544.ipfs.nftstorage.link/,true
+SOUL,SOUL,F6weWmuc1vwdL4u38Ro9jKXHEMjP9BoNdWMF5o5TvtJf,9,https://rkmp2iorabaal3oqws5jj233ruddww4itlhmkrigin5nctwm4isa.arweave.net/ipj9IdEAQAXt0LS6lOt7jQY7W4iazsVFBkN60U7M4iQ,true
```
Comment (Timoon21) [inline validated-tokens.csv:756]:
```suggestion
SOUL,SOUL,F6weWmuc1vwdL4u38Ro9jKXHEMjP9BoNdWMF5o5TvtJf,9,https://arweave.net/ipj9IdEAQAXt0LS6lOt7jQY7W4iazsVFBkN60U7M4iQ,true
```

**Review by Timoon21 — CHANGES_REQUESTED** [review]

**Review by Timoon21 — DISMISSED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] Community validation

File: validated-tokens.csv (hunk 1 above)
Comment (iskeceli) [reply validated-tokens.csv]: updated

**Comment by ArashiSol** [comment]
@meijiao

**Pull request merged**